lib/access_import/journal_transform.rb

```diff
@@ -0,0 +1,35 @@
+require 'acc_utils'
+
+class JournalTransform
+  include AccUtils
+
+  def initialize(*accessors)
+    accessors.each do |accessor|
+      instance_variable_set("@#{accessor.class.name.underscore}", accessor)
+    end
```

Comment:
This doesn't look like it's actually used?

Reply:
Right. Ill make it by inheritance. It's the same in all those parameter preparers